src/pyedb/dotnet/edb_core/edb_data/raptor_x_simulation_setup_data.py

```diff
@@ -0,0 +1,448 @@
+        self._advanced_settings = edb_setup_info.SimulationSettings.AdvancedSettings
+
+    @property
+    def auto_removal_sliver_poly(self):
+        return self._advanced_settings.AutoRemovalSliverPoly
+
+    @auto_removal_sliver_poly.setter
+    def auto_removal_sliver_poly(self, value):
+        self._advanced_settings.AutoRemovalSliverPoly = EdbValue(value).tofloat
+
+    @property
+    def cell_per_wave_length(self):
+        """This setting describes the number of cells that fit under each wavelength. The wavelength is
+        calculated according to the Max Frequency or the Mesh Frequency, unless specified by user through
+        this setting. E.g. Setting Cells/Wavelength to 20 means that an object will be divided into 10 cells
+        if its width or length is 1/2 wavelengths.
+        Units: unitless.
+        """
+        return self._advanced_settings.CellsPerWavelength
+
+    @cell_per_wave_length.setter
+    def cell_per_wave_length(self, value):
+        if isinstance(value, int):
+            self._advanced_settings.CellsPerWavelength = value
+
+    @property
+    def edge_mesh(self):
+        """This option controls both, the thickness and the width of the exterior conductor filament.
+        When specified, it prevails over the Mesh Frequency or Max Frequency during mesh calculation.
+        Example: "0.8um".
+        """
+        return self._advanced_settings.EdgeMesh
+
+    @edge_mesh.setter
+    def edge_mesh(self, value):
+        self._advanced_settings.EdgeMesh = EdbValue(value).tostring
+
+    @property
+    def eliminate_slit_per_hole(self):
+        """This is a setting that internally simplifies layouts with strain relief or thermal relief slits and
+        holes. It will examine each hole separately against the whole polygon it belongs to.
+        If the area of the hole is below the threshold defined in this setting, then the hole will be filled.
+        Units: unitless.
+        """
+        return self._advanced_settings.EliminateSlitPerHoles
+
+    @eliminate_slit_per_hole.setter
+    def eliminate_slit_per_hole(self, value):
+        self._advanced_settings.EliminateSlitPerHoles = EdbValue(value).tofloat
+
+    @property
+    def mesh_frequency(self):
+        """User can override the default meshing applied by setting a custom frequency for mesh generation.
+        Example: "1GHz".
+        """
+        return self._advanced_settings.MeshFrequency
+
+    @mesh_frequency.setter
+    def mesh_frequency(self, value):
+        self._advanced_settings.MeshFrequency = EdbValue(value).tostring
+
+    @property
+    def net_settings_options(self):
+        """A list of Name, Value pairs that stores advanced option."""
+        return [val for val in list(self._advanced_settings.NetSettingsOptions)]
+
+    @net_settings_options.setter
+    def net_settings_options(self, value):
+        if isinstance(value, list):
+            self._advanced_settings.NetSettingsOptions = convert_py_list_to_net_list(value)
```

Comment:
Can you log an error if value is not list ?

Reply:
done